core/math/math_funcs.h

```diff
@@ -215,15 +215,20 @@ class Math {
 	static _ALWAYS_INLINE_ double lerp(double p_from, double p_to, double p_weight) { return p_from + (p_to - p_from) * p_weight; }
 	static _ALWAYS_INLINE_ float lerp(float p_from, float p_to, float p_weight) { return p_from + (p_to - p_from) * p_weight; }
 
+	static _ALWAYS_INLINE_ double angle_diff(double minuend, double subtrahend) {
+		double diff = fposmod(minuend - subtrahend, (double)Math_TAU);
+		return fmod(2.0 * diff, Math_TAU) - diff;
+	}
+	static _ALWAYS_INLINE_ float angle_diff(float minuend, float subtrahend) {
```

Comment:
1. `minuend` and `subtrahend` are (IMO) bad names for multiple reasons, including the fact that they are not commonly used words in English and that they are not descriptive of the purpose of the variables.

2. "In" parameters need to be prefixed with `p_` in C++.

Reply:
changing `minuend` to `to` and `subtrahend` to `from` seems good?
Sorry for not noticing `p_`, are they applied to all parameters exposed through gdscript?

Reply:
The `p_` prefix is used for all parameters in all of Godot's code, to differentiate them from class variables on one hand, and from return parameters (`r_`) on the other.